flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisReadOptions.java

```diff
@@ -121,52 +126,20 @@ public void setFilterQuery(String filterQuery) {
         this.filterQuery = filterQuery;
     }
 
-    public static Builder builder() {
-        return new Builder();
+    public boolean getUseFlightSql() {
+        return useFlightSql;
     }
 
-    public static DorisReadOptions defaults() {
-        return DorisReadOptions.builder().build();
+    public String getFlightSqlPort() {
+        return flightSqlPort;
     }
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        DorisReadOptions that = (DorisReadOptions) o;
-        return useOldApi == that.useOldApi
-                && Objects.equals(readFields, that.readFields)
-                && Objects.equals(filterQuery, that.filterQuery)
-                && Objects.equals(requestTabletSize, that.requestTabletSize)
-                && Objects.equals(requestConnectTimeoutMs, that.requestConnectTimeoutMs)
-                && Objects.equals(requestReadTimeoutMs, that.requestReadTimeoutMs)
-                && Objects.equals(requestQueryTimeoutS, that.requestQueryTimeoutS)
-                && Objects.equals(requestRetries, that.requestRetries)
-                && Objects.equals(requestBatchSize, that.requestBatchSize)
-                && Objects.equals(execMemLimit, that.execMemLimit)
-                && Objects.equals(deserializeQueueSize, that.deserializeQueueSize)
-                && Objects.equals(deserializeArrowAsync, that.deserializeArrowAsync);
```

Comment:
Is it possible to directly add the two attributes to the equals and hashcode methods?

Reply:
ok